workflows/cellranger-quant/run-cellranger.nf

```diff
@@ -29,15 +30,17 @@ process cellranger{
   output:
     path output_id
   script:
-    output_id = "${id}-${index}"
+    output_id = "${id}-${index}-${params.include_introns ? '--include-introns' : 'pre_mRNA'}"
```

Comment:
I assume you want to distinguish here with something like `pre_mRNA` vs `mRNA`? (Not sure what the appropriate alternative is... Maybe just `pre_mRNA` vs nothing? In any case, I think this needs to be updated to reflect the chosen option accurately. Maybe: 
```suggestion
    output_id = "${id}-${index}${params.include_introns ? '-pre_mRNA' : ''}"
```

Or if we want to name both versions explicitly:
```suggestion
    output_id = "${id}-${index}-${params.include_introns ? 'pre_mRNA' : 'mRNA'}"
```